orc8r/cloud/docker/docker-compose.cwf.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+# NOTE: x-orc8r is somewhat replicated from the main docker-compose file.
+# Please keep it in sync.
+x-orc8r_service: &orc8r_service
+  image: orc8r_base
+  environment:
+    TEST_MODE: "1"  # Used to run dev scripts on startup
+    SERVICE_HOSTNAME: localhost  # All services are reachable through localhost
+    SERVICE_REGISTRY_MODE: yaml  # YAML backed service registry
```

Comment:
"Yaml" registry mode doesn't work with the containers split up this way does it?  Services will look for each other at localhost:port but given we're not running with host networking. If this is true we should use "docker" mode.